torchvision/datapoints/_dataset_wrapper.py

```diff
@@ -97,6 +96,10 @@ def wrap_dataset_for_transforms_v2(dataset, target_keys=None):
             f"but got {target_keys}"
         )
 
+    return _make_wrapped_dataset(dataset, target_keys)
+
+
+def _make_wrapped_dataset(dataset, target_keys):
```

Comment:
This needs to be a standalone function now, since the output of `__reduce__` needs to be pickleable itself.

Reply:
I don't really get it.
What's the part in `wrap_dataset_for_transforms_v2()` that can't be pickled?